### PR TITLE
Move to bintray

### PR DESCRIPTION
**Pull request by aforcier**
cc @maxme

### CODE REVIEW

File: WordPressGraphView/build.gradle [hunk 1]
```diff
@@ -1,83 +1,44 @@
...
-        }
-    }
+publish {
+    artifactId = 'graphview'
+    userOrg = 'wordpress-mobile'
+    groupId = 'org.wordpress'
+    uploadName = 'graphview'
+    description = 'Android library for programmatic visual diagram creation'
+    publishVersion = android.defaultConfig.versionName
+    licences = ['GPL-2.0']
```
Comment (maxme) [inline WordPressGraphView/build.gradle]: Licence is LGPL: https://github.com/wordpress-mobile/GraphView/blob/develop/license.txt

Comment (aforcier) [reply WordPressGraphView/build.gradle]: Ack. Looks like the old license for sonatype was wrong too. Should I push a new version with the updated license?

Comment (aforcier) [reply WordPressGraphView/build.gradle]: Fixed the license part in 5fe24ee.

Comment (aforcier) [reply WordPressGraphView/build.gradle]: Deleted and re-uploaded `3.4.1` with the fixed license.

**Comment by maxme** [comment]
:shipit:

**Pull request merged**